core/plans/microvm_create_update.go

```diff
@@ -66,6 +66,13 @@ func (p *microvmCreateOrUpdatePlan) Create(ctx context.Context) ([]planner.Proce
 		return nil, fmt.Errorf("adding root dir step: %w", err)
 	}
 
+	// MicroVM provider doesn't have virtiofs
+	if provider.Capabilities().Has(models.VirtioFSCapability) {
```

Comment:
We need to make sure that we have the earlier validation check to fail early when the command first executes.